### PR TITLE
Update stubutil.py is_c_module with .pyd check

### PR DESCRIPTION
**Pull request by AlexArcPy**
Added check for .pyd file extension used on Windows

### CODE REVIEW

File: mypy/stubutil.py [hunk 1]
```diff
@@ -87,7 +87,8 @@ def find_unique_signatures(sigs: Sequence[Sig]) -> List[Sig]:
 
 
 def is_c_module(module: ModuleType) -> bool:
-    return '__file__' not in module.__dict__ or module.__dict__['__file__'].endswith('.so')
+    return '__file__' not in module.__dict__ or any(module.__dict__['__file__'].endswith(
+      ext) for ext in ['.so', '.pyd'])
```
Comment (gvanrossum) [inline mypy/stubutil.py]: This is causing the linter to fail:
```
./mypy/stubutil.py:91:7: E121 continuation line under-indented for hanging indent
```
You can run the linter locally: `flake8 mypy/stubutil.py`

Though honestly I would refactor the code a little more -- the `any()` feels like it makes the code more complex than it needs to be. Maybe use `os.path.splitext()`?

**Comment by AlexArcPy** [comment]
Thanks, Guido! I did not dare to refactor, but thanks for the hint, it is done, the linter should be happy now. Please review the latest changes and let me know if there is anything more that should be fixed.

**Comment by gvanrossum** [comment]
Thanks Alex! That was actually exactly the refactoring I was thinking of. :-)

**Comment by mikiboii** [comment]
i dont understand how didi u generate pyi file for pyd python file

**Pull request merged**